src/sql/parts/notebook/notebook.css

```diff
@@ -83,4 +83,10 @@
 }
 .moreActions.actionhidden {
 	visibility: hidden
+}
+
+.notebookEditor .notebook-cellTable {
+	border: 1px silver solid;
```

Comment:
Shouldn't this be in notebookStyles.ts and use a theme color instead of silver? How does this work for other grid implementations, what theme color do they use? Is this really hardcoded? That's a real shame, but OK....

Reply:
It's really hardcoded where we use SlickGrid :(. Take a look at slick.grid.css.